apikeyAuthorizerV2/src/test/mocks.js

```diff
@@ -10,7 +10,12 @@ const mockIamPolicyOk = {
             }
         ]
     },
-    context: {},
+    context: {
+        "uid": "APIKEY-testApiKey",
+        "cx_id": "cxId",
+        "cx_groups": '',
```

Comment:
C'é un caso di test anche con i gruppi?

Reply:
Attualmente i gruppi che vengono estratti dalla tabella delle apikey non subiscono elaborazioni. Alla fine del processo, qualora siano presenti, vengono semplicemente concatenati in una stringa nel context.cx_groups, quindi non ci sono test che fanno asserzioni sui gruppi. Ritieni sia necessario aggiungerli?

Reply:
Beh, anche solo vedere che vengono passati. Se mettiamo un array vuoto non sappiamo se vengono poi passati o c'è una lista vuota perché manca il codice che la valorizza. No?

Reply:
Ok, ho aggiunto qualche asserzione nei test per controllare anche i gruppi restituiti.